tests/manager/manager-debug-001.phpt

```diff
@@ -10,16 +10,18 @@ require_once __DIR__ . "/../utils/basic.inc";
 $name = tempnam(sys_get_temp_dir(), "PHONGO");
 unlink($name);
 mkdir($name);
-$manager = new MongoDB\Driver\Manager(STANDALONE, array(), array("debug" => $name));
```

Comment:
This was changed for [PHPC-704](https://jira.mongodb.org/browse/PHPC-704), since we no longer want to support INI modifications via a constructor argument. As a result, I need to manually enable/disable the INI setting below.